com.rawtherapee.RawTherapee.yaml

```diff
@@ -230,33 +230,11 @@ modules:
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
     sources:
       - type: archive
-        url: https://github.com/Beep6581/RawTherapee/releases/download/5.7/rawtherapee-5.7.tar.xz
-        sha256: dbd7c7cf7488fb97c520821eee2c745291637644b391e3ec0ed3a29701f1a9c7
-      # Update XDG files
+        url: https://github.com/Beep6581/RawTherapee/releases/download/5.8/rawtherapee-5.8.tar.xz
+        sha256: 360528a0aae922eb5af8742f415475fb91b6d62a739da5f2804828f04ec40853
       # Update screenshots in appdata file
-      # Add developer_name tag
```

Comment:
It isn't upstreamed.

Reply:
this is why one patch per patch file is nice. Because we see better.